--- /Users/dids/.clobber/src/edk2/Clover/CloverPackage/package/buildpkg.sh	2019-07-19 11:41:21.000000000 +0300
+++ /Users/dids/.clobber/src/edk2/Clover/CloverPackage/package/buildpkg_patched.sh	2019-07-19 11:42:11.000000000 +0300
@@ -13,6 +13,15 @@
 set -u  # Abort with unset variables
 #set -x
 
+isDefaultPackage () {
+    local packageRefId="${1}"
+    if [[ $packageRefId =~ ^(.*)(apfsdriverloader|aptiomemoryfix|hfsplus)(.*)$ ]]; then
+        return 0
+    else
+        return 1
+    fi
+}
+
 usage () {
 printf "\n\e[1m%s\e[0m" "Usage: $0 --srcroot <path> --symroot <name> --builddir <path> [flag1 flag2...]"
 echo
@@ -526,7 +535,7 @@
                        --subst="INSTALLER_CHOICE=$packageRefId" MarkChoice
     buildpackage "$packageRefId" "${choiceId}" "${PKG_BUILD_DIR}/${choiceId}" "/EFIROOTDIR"
     if [[ ${NOEXTRAS} != *"CloverEFI"* ]]; then
-        addChoice --start-visible="true" --start-selected="choicePreviouslySelected('$packageRefId')"  \
+        addChoice --start-visible="true" --start-selected="cloverPackageFirstRun() || choicePreviouslySelected('$packageRefId')"  \
                   --pkg-refs="$packageRefId" "${choiceId}"
     else
         addChoice --start-visible="false" --start-selected="true"  \
@@ -840,12 +849,22 @@
         # (i.e. Clover package never run on that target partition).
         # Otherwise each single choice start selected only for legacy Clover and only if you previously selected it
         buildpackage "$packageRefId" "${driverChoice}" "${PKG_BUILD_DIR}/${driverChoice}" "${driverDestDir}"
-        addChoice --group="Recommended64" \
+        if isDefaultPackage "$packageRefId"; then
+            addChoice --group="Recommended64" \
+                  --start-visible="true" \
+                  --enabled="!choices['UEFI.only'].selected" \
+                  --start-selected="!choices['UEFI.only'].selected"  \
+                  --visible="!choices['UEFI.only'].selected"     \
+                  --pkg-refs="$packageRefId"  "${driverChoice}"
+        else
+            addChoice --group="Recommended64" \
                   --start-visible="true" \
                   --enabled="!choices['UEFI.only'].selected" \
-                  --start-selected="!choices['UEFI.only'].selected &amp;&amp; (cloverPackageFirstRun() || choicePreviouslySelected('$packageRefId'))"  \
+                  --start-selected="!choices['UEFI.only'].selected &amp;&amp; (choicePreviouslySelected('$packageRefId'))"  \
                   --visible="!choices['UEFI.only'].selected"     \
                   --pkg-refs="$packageRefId"  "${driverChoice}"
+        fi
+        
         rm -R -f "${PKG_BUILD_DIR}/${driverChoice}"
     done
 fi
@@ -1007,10 +1026,17 @@
         fi
 
         buildpackage "$packageRefId" "${driverChoice}" "${PKG_BUILD_DIR}/${driverChoice}" "${driverDestDir}"
-        addChoice --group="Recommended64UEFI" \
+        if isDefaultPackage "$packageRefId"; then
+            addChoice --group="Recommended64UEFI" \
                   --start-visible="true" \
-                  --start-selected="cloverPackageFirstRun() || choicePreviouslySelected('$packageRefId')" \
+                  --start-selected="true" \
                   --pkg-refs="$packageRefId"  "${driverChoice}"
+        else
+            addChoice --group="Recommended64UEFI" \
+                  --start-visible="true" \
+                  --start-selected="choicePreviouslySelected('$packageRefId')" \
+                  --pkg-refs="$packageRefId"  "${driverChoice}"
+        fi
         rm -R -f "${PKG_BUILD_DIR}/${driverChoice}"
     done
 fi
@@ -1359,7 +1385,6 @@
     for (( i = 0 ; i < ${#specialThemes[@]} ; i++ )); do
         local themeName=${specialThemes[$i]##*/}
         # Don't add christmas and newyear themes if month < 11
-        [[ $currentMonth -lt 11 ]] && [[ "$themeName" == christmas || "$themeName" == newyear ]] && continue
         mkdir -p "${PKG_BUILD_DIR}/${themeName}/Root/"
         rsync -r --exclude=.svn --exclude="*~" "$artwork/${specialThemes[$i]}/" "${PKG_BUILD_DIR}/${themeName}/Root/${themeName}"
         packageRefId=$(getPackageRefId "${packagesidentity}" "${themeName}")
@@ -1369,7 +1394,7 @@
                            InstallTheme
 
         buildpackage "$packageRefId" "${themeName}" "${PKG_BUILD_DIR}/${themeName}" "${themeDestDir}"
-        addChoice --start-visible="false"  --start-selected="true"  --pkg-refs="$packageRefId" "${themeName}"
+        addChoice --group="Themes"  --start-visible="true"  --start-selected="choicePreviouslySelected('$packageRefId')"  --pkg-refs="$packageRefId" "${themeName}"
     done
 fi
 
