--- buildpkg.sh
+++ buildpkg.sh
@@ -1109,7 +1109,10 @@
                             "VirtualSMC"
         fi
         buildpackage "$packageRefId" "${driverChoice}" "${PKG_BUILD_DIR}/${driverChoice}" "${driverDestDir}"
-        addChoice --group="Drivers64UEFI" --start-visible="true" --start-selected="true" --pkg-refs="$packageRefId"  "${driverChoice}"
+        ###   CLOBBER PATCH START   ###
+        #addChoice --group="Drivers64UEFI" --start-visible="true" --start-selected="true" --pkg-refs="$packageRefId"  "${driverChoice}"
+        addChoice --group="Drivers64UEFI" --start-visible="true" --start-selected="choicePreviouslySelected('$packageRefId')" --pkg-refs="$packageRefId"  "${driverChoice}"
+        ###    CLOBBER PATCH END    ###
         rm -R -f "${PKG_BUILD_DIR}/${driverChoice}"
     done
 fi
