--- Clover.dsc
+++ Clover.dsc
@@ -352,7 +352,9 @@
   #EmbeddedPkg/Universal/MmcDxe/MmcDxe.inf
   #Clover/OsxMmcDxe/MmcDxe.inf
   Clover/FileSystems/FatPkg/EnhancedFatDxe/Fat.inf
-  Clover/FileSystems/ApfsDriverLoader/ApfsDriverLoader.inf
+  ### CLOBBER PATCH START ###
+  # Clover/FileSystems/ApfsDriverLoader/ApfsDriverLoader.inf
+  ### CLOBBER PATCH END   ###
   # FS from grub
 !ifndef NO_GRUB_DRIVERS
   Clover/FileSystems/GrubFS/src/EXFAT.inf
