--- ebuild.sh
+++ ebuild.sh
@@ -801,7 +801,12 @@
                                   'https://github.com/acidanthera/AppleSupportPkg'
                                   'https://github.com/CupertinoNet/CupertinoModulePkg'
                                   'https://github.com/CupertinoNet/EfiMiscPkg'
-                                  'https://github.com/CupertinoNet/EfiPkg')
+                                  'https://github.com/CupertinoNet/EfiPkg'
+    ###   CLOBBER PATCH START   ###
+                                  #'https://github.com/CupertinoNet/EfiPkg')
+                                  'https://github.com/CupertinoNet/EfiPkg'
+                                  'https://github.com/acidanthera/OcSupportPkg')
+    ###   CLOBBER PATCH END   ###
     # add below drivers you want to build
     local externalDrivers=( AptioFixPkg AppleSupportPkg )
 
@@ -824,7 +829,10 @@
         local branch=master
 
         case $pkg in
-        CupertinoModulePkg | EfiMiscPkg | EfiPkg)
+        ###   CLOBBER PATCH START   ###
+        #CupertinoModulePkg | EfiMiscPkg | EfiPkg)
+        CupertinoModulePkg | EfiMiscPkg)
+        ###   CLOBBER PATCH END   ###
           branch=development
         ;;
         esac
