--- ebuild.sh
+++ ebuild.sh
@@ -799,8 +799,8 @@
     # add github links below to checkout packages
     local extDriversDependecies=( 'https://github.com/acidanthera/AptioFixPkg'
                                   'https://github.com/acidanthera/AppleSupportPkg'
+                                  'https://github.com/acidanthera/OcSupportPkg'
                                   'https://github.com/CupertinoNet/CupertinoModulePkg'
-                                  'https://github.com/CupertinoNet/EfiMiscPkg'
                                   'https://github.com/CupertinoNet/EfiPkg')
     # add below drivers you want to build
     local externalDrivers=( AptioFixPkg AppleSupportPkg )
@@ -824,7 +824,7 @@
         local branch=master
 
         case $pkg in
-        CupertinoModulePkg | EfiMiscPkg | EfiPkg)
+        CupertinoModulePkg)
           branch=development
         ;;
         esac
